CHANGELOG.next.asciidoc

```diff
@@ -200,6 +200,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add optional regex based cid extractor to `add_kubernetes_metadata` processor. {pull}17360[17360]
 - Add `urldecode` processor to for decoding URL-encoded fields. {pull}17505[17505]
 - Add support for AWS IAM `role_arn` in credentials config. {pull}17658[17658] {issue}12464[12464]
+- When using the `decode_json_fields` processor, decoded fields are now deep-merged into existing event. {pull}17958[17958]
```

Comment:
How about: `Change `decode_json_fields` to merge parsed json objects with existing objects in the event instead of fully replacing them.`.

Would it make sense to treat this as a bug and add the changelog note to the bugfix section?

Reply:
Oh, I misunderstood https://github.com/elastic/beats/pull/17958#discussion_r414556011. Sure, I will note this as a bug but only in the scope of Filebeat (not a generic bug in Beats).

Reply:
Done in b7304af16.